SimpleWAWS/Global.asax.cs

```diff
@@ -181,6 +195,19 @@ protected void Application_AuthenticateRequest(Object sender, EventArgs e)
                     SecurityManager.HandleAnonymousUser(context);
                 }
             }
+            else //coming in from auth provider
```

Comment:
Yes. In the functions flow, for the first time when the redirect happens from AuthProvider , `TryAuthenticateSessionCookie` returns a true, but we dont want people to stay on TryAppService, but instead go on to TryFunctions.  The comment just means we got redirected , after successful login from the AuthProvider, now lets redirect to `TryFunctions` if we need to.